packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md

```diff
@@ -2,138 +2,98 @@
 
-Yes. One of the key benefits of developing a pre-built integration with Okta is that you can leverage our existing active directory support. By integrating with Okta (for single sign-on and provisioning), you effectively have the ability to integrate with your customer's on-premise AD or LDAP infrastructure for authentication (log into your cloud app with their corporate password), authorization (use details like AD groups to drive access rights), and provisioning policies.
+Have you submitted an app but haven't seen a change in the review status in the [OIN Manager](https://oinmanager.okta.com)? Send an email to <developers@okta.com>.
 
-**What is Secure Web Authentication (SWA)?**
+**Q: I am an ISV having issues integrating my app or have questions about single sign-on support in my app. How do I contact Okta?**
 
-SWA was developed by Okta to provide single sign-on for apps that do not support federated sign-on methods. Users can still sign in directly through the application and then enter their credentials for these apps on their Okta homepage. These credentials are stored so that users can access their apps with a single sign-on. When users first sign on to a SWA app from their homepage, they see a pop-up message asking if they were able to sign on successfully.
+If you have any technical questions, you can submit them to <developers@okta.com> or post your questions on our [Developer Forum](https://devforum.okta.com/).
 
-**What is the process after I have submitted my app using the OIN Manager?**
+**Q: I am an ISV whose app is already listed in the OIN. How do I request changes or updates to the existing integration?**
 
-After you submit your app integration, it goes through a review by the Okta OIN Operations team, which reviews and tests your integration. We will reach out to you directly if your integration does not pass the review, if we need you to resubmit, or if we need additional information. Following the completed review, the integration is promoted to the OIN. Typical review time is two weeks, and you can track the progress of your submission in the [OIN Manager](https://oinmanager.okta.com).
+Submit your application with the changes as you would if your application were not listed in the OIN. After your submission goes under review by Okta, we compare your submission against the existing customer-facing version, and update it accordingly. If there are special instructions you'd like to provide to us, add them in the Additional Instructions text box on the submission form. We will reach out to you directly with any questions.
```

Comment:
Additional Instructions should be bold, and the word "text" isn't necessary.

Reply:
Fixes made.